gpg-client-wrapper

```diff
@@ -5,10 +5,56 @@ eoo=0       # end of options reached
 output=0    # do we try to write to file
 target=()   # where do we try to write to
 fd_for_stdout=63 # some (hopefuly) unused FD, later redirected to stdout
+localgpg=0 #use local gpg (for ex --gen-rand etc.)
+origargs="$@"
 
 while [[ $1 ]]; do
     if ! ((eoo)); then
         case "$1" in
+            #when those arguments are present will not use the keyring, and so they can be executed with local gpg
+            # can be used in combination with sign
+            #-c)
+            #    localgpg=1
+            #    break
+            #;;
+            --passprase-fd)
```

Comment:
This alone should not trigger local gpg usage.

Reply:
removed that one, it was misspelled anyway